tests/conftest.py

```diff
@@ -52,64 +140,21 @@ def local_file_collections(test_data_path):
 
 @pytest.fixture
 def local_expected_csvs(test_data_path):
-    with open(test_data_path / "expected/assessments.csv", "r") as f:
-        assessments = list(csv.DictReader(f))
-    with open(test_data_path / "expected/users.csv", 'r') as f:
-        users = list(csv.DictReader(f))
-    with open(test_data_path / "expected/grades.csv", "r") as f:
-        grades = list(csv.DictReader(f))
-    with open(test_data_path / "expected/enrollments.csv", 'r') as f:
-        enrolments = list(csv.DictReader(f))
-    with open(test_data_path / "expected/courses.csv", 'r') as f:
-        courses = list(csv.DictReader(f))
-    with open(test_data_path / "expected/quiz_questions.csv", 'r') as f:
-        quiz_questions = list(csv.DictReader(f))
-    with open(
-        test_data_path / "expected/quiz_question_contents.csv", 'r'
-    ) as f:
-        quiz_question_contents = list(csv.DictReader(f))
-    with open(
-        test_data_path / "expected/quiz_multichoice_answers.csv", 'r'
-    ) as f:
-        quiz_multichoice_answers = list(csv.DictReader(f))
-    with open(test_data_path / "expected/ib_input_instances.csv", 'r') as f:
-        ib_input_instances = list(csv.DictReader(f))
-    with open(test_data_path / "expected/ib_pset_problems.csv", 'r') as f:
-        ib_pset_problems = list(csv.DictReader(f))
-    with open(test_data_path / "expected/course_contents.csv", 'r') as f:
-        course_contents = list(csv.DictReader(f))
-    with open(test_data_path / "expected/quiz_attempts.csv", 'r') as f:
-        quiz_attempts = list(csv.DictReader(f))
-    with open(test_data_path /
-              "expected/quiz_attempt_multichoice_responses.csv", 'r') as f:
-        quiz_attempt_multichoice_responses = list(csv.DictReader(f))
-    with open(test_data_path / "expected/content_loads.csv", 'r') as f:
-        content_loads = list(csv.DictReader(f))
-    with open(test_data_path / "expected/ib_pset_problem_attempts.csv", 'r')\
-            as f:
-        ib_pset_problem_attempts = list(csv.DictReader(f))
-    with open(test_data_path / "expected/ib_input_submissions.csv", 'r') as f:
-        ib_input_submissions = list(csv.DictReader(f))
-    return (
-        assessments,
-        users,
-        grades,
-        enrolments,
-        courses,
-        quiz_questions,
-        quiz_question_contents,
-        quiz_multichoice_answers,
-        ib_input_instances,
-        ib_pset_problems,
-        course_contents,
-        quiz_attempts,
-        quiz_attempt_multichoice_responses,
-        content_loads,
-        ib_pset_problem_attempts,
-        ib_input_submissions
-    )
+    return open_and_read_expected_csvs(test_data_path, 'expected')
+
+
+@pytest.fixture
+def local_expected_filtered_csvs(test_data_path):
+    return open_and_read_expected_csvs(test_data_path, 'expected_filtered')
 
 
 @pytest.fixture
 def autogenerated_user_uuid():
     return UUID("7080c78d-298b-40ba-a68d-55d6a93b00fb")
+
+
+@pytest.fixture
+def local_course_filter(test_data_path):
```

Comment:
Minor: Similar naming suggestion around perhaps calling this fixture `local_research_filter`.